src/pokedex_plus_hgss.c

```diff
@@ -4370,40 +4370,25 @@ static void SetSpriteInvisibility(u8 spriteArrayId, bool8 invisible)
 #define TYPE_ICON_PAL_NUM_0     13
 #define TYPE_ICON_PAL_NUM_1     14
 #define TYPE_ICON_PAL_NUM_2     15
```

Comment:
These palette defines are not used in the Type struct, where raw values are used. Maybe it'd be of interest to use them there as well? If not, I would just remove them here.